### ISSUE TITLE
Documents attached to literature requests, borrowing request, acquisition orders

### ISSUE THREAD

**Issue by agentilb** [CERNDocumentServer/cds-migrator-kit#157]
Would it be possible to be able to attach those requests to Journals (Series - Type periodical)?

**Comment by kprzerwa** [comment]
no, attaching the requests to the series types is not possible. Could you describe why would you need this functionality ?
